docs/Guide/plugins/plugin-i18next.mdx

```diff
@@ -2,4 +2,337 @@
 title: Plugin i18next
 ---
 
-## TODO
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+An implementation of i18next's [filesystem backend] for Sapphire. It allows you to use a JSON-based `languages`
+directory to add internationalization for your bot using `SapphireClient`'s `fetchLanguage` hook and a custom message
+extension, adding features such as `sendTranslated` and `resolveKey`.
+
+[filesystem backend]: https://github.com/i18next/i18next-fs-backend
+
+## Installation
+
+```bash npm2yarn2pnpm
+npm install @sapphire/plugin-i18next @sapphire/framework discord.js
+```
+
+## Usage
+
+This registers the methods and options necessary for message translations in the Sapphire client.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript
+// Main bot file
+// Be sure to register the plugin before instantiating the client.
+require('@sapphire/plugin-i18next/register');
+```
+
+</TabItem>
+<TabItem value="esm" label="ESM">
+
+```javascript
+// Main bot file
+// Be sure to register the plugin before instantiating the client.
+import '@sapphire/plugin-i18next/register';
+```
+
+</TabItem>
+<TabItem value="typescript" label="TypeScript">
+
+```typescript
+// Main bot file
+// Be sure to register the plugin before instantiating the client.
+import '@sapphire/plugin-i18next/register';
+```
+
+</TabItem>
+</Tabs>
+
+The basic structure of a translation file is as follows:
+
+```javascripton
+// languages/en-US/commands/ping.json
+{
+	"success": "Pong!",
+	"success_with_args": "Pong! Took me {{latency}}ms to reply"
+}
+```
+
+The `resolveKey` function can be used anywhere to get translated text by its key. In this example, it is used in a
+method to send a message.
+
+<Tabs groupId="language-choice">
+<TabItem value="javascript" label="JavaScript" default>
+
+```javascript
+const { resolveKey } = require('@sapphire/plugin-i18next');
```

Comment:
Another consistency thing that drives me a bit mad throughout the guides & docs is tab indents vs two space indents :')

Reply:
For the record: everything should be 2 space indents because otherwise it renders like crap in browsers.

Reply:
And I always format the code so I see no issue with spaces and tabs